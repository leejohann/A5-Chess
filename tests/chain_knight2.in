move 1 7 2 5
move 1 0 2 2
move 2 5 0 4
move 2 2 4 3
quit
