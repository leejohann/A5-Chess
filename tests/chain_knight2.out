RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ n _ _ 
pppppppp
r bqkbnr

it is Black's turn
R_BQKBNR
PPPPPPPP
 _N_ _ _
_ _ _ _ 
 _ _ _ _
_ n _ _ 
pppppppp
r bqkbnr

it is White's turn
R_BQKBNR
PPPPPPPP
 _N_ _ _
_ _ _ _ 
n_ _ _ _
_ _ _ _ 
pppppppp
r bqkbnr

it is Black's turn
R_BQKBNR
PPPPPPPP
 _ _ _ _
_ _ N _ 
n_ _ _ _
_ _ _ _ 
pppppppp
r bqkbnr

it is White's turn

